Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:10
+# Use alpine node version
+FROM node:10.15.3-alpine
 
-# Create app directory
-WORKDIR /usr/local
+# Create working directory and set node user as owner
+WORKDIR /usr/src/app
+RUN chown node:node /usr/src/app
```

Comment:
Set the node user as owner of the working directory to ensure there will be no unexpected permission issues.